### DISCUSSION TITLE
Cosine similarity

### DISCUSSION THREAD

**Discussion by ceiron61** [nuno-agostinho/cTRAP#5] [Ideas]
Hello and thanks for a very nice package.

Would you consider including cosine similarity as a fourth method alongside GSEA, Pearson and Spearman correlation?

**Comment by nuno-agostinho** [comment]
Hey @enricoferrero, I am glad you like the package.

> Would you consider including cosine similarity as a fourth method alongside GSEA, Pearson and Spearman correlation?

What would be the advantages of using cosine similarity over the other methods?
